src/data_standardization/tests/test_standardize_point_data.py

```diff
@@ -139,14 +138,88 @@ def test_read_file_info_agg(tmpdir):
         },
         'feat_agg': 'latest',
         'value': 87081
-        },
+    },
         {
-        'feature': 'aggtest',
-        'date': '2012-05-01T20:00:00-04:00',
-        'location': {
-            'latitude': 39.90415873,
-            'longitude': -75.19348751
-        },
-        'feat_agg': 'latest',
-        'value': 87679
+            'feature': 'aggtest',
```

Comment:
indentation here also looks weird

Reply:
Huh, this one seems fine to me? Indent within brackets?